examples/featureFlagSampleWithAppConfig.mjs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as dotenv from "dotenv";
+dotenv.config()
+
+import { load } from "@azure/app-configuration-provider";
+const connectionString = process.env.APPCONFIG_CONNECTION_STRING;
+const appConfig = await load(connectionString, {
+    featureFlagOptions: {
+        enabled: true,
+        selectors: [{
+            keyFilter: "*"
+        }],
+        refresh: {
+            enabled: true
+        }
+    }
+});
+
+import { ConfigurationMapFeatureFlagProvider, FeatureManager } from "@microsoft/feature-management";
```

Comment:
Can you put the package import statement together at the beginning of the file?

Reply:
Same reason https://github.com/microsoft/FeatureManagement-JavaScript/pull/24#discussion_r1729158929